en/04_Changelogs/5.0.0.md

```diff
@@ -62,6 +63,29 @@ This is a major release and as a result there are a number of breaking API chang
     - `SilverStripe\ORM\Connect\Query`
   - Removed `SilverStripe\ORM\DataList::getGenerator()` (use `getIterator()` instead)
   - Removed the `SilverStripe\ORM\Map_Iterator` class. `SilverStripe\ORM\Map` now uses a generator instead.
+- Removed various API related to `SwiftMailer`
+  - Removed the following classes and interfaces
+    - `SilverStripe\Control\Email\Mailer`
+    - `SwiftMailer`
+    - `SwiftPlugin`
+  - Removed the following methods from `SilverStripe\Control\Email\Email`
+    - Replaced with `Email::getData()`
+      - `BaseURL()`
+      - `IsEmail()`
+    - Has equivalent functionality via `Email::getTextBody()`
+      - `hasPlainPart()`
+      - `findPlainPart()`
```

Comment:
We discussed in https://github.com/silverstripe/silverstripe-framework/pull/10517#discussion_r984063507 that these were removed without equivalent functionality - which is what the deprecation messages say.

If you're now saying they _do_ have equivalent functionality, please create a new PR targetting `4` with an updated deprecation message. Otherwise, please indicate these as not having equivalent functionality.

Reply:
Looking at the code in https://github.com/symfony/mime/blob/6.1/Email.php the docs here are correct, so we need to fix those deprecations.

Reply:
I've moved `hasPlainPart()` and `findPlainPart()` to under the `Not replaced with equivalent functionality` section.  While they're kind of sort of replaced with `getTextBody()`, it's not really the same thing